Update stations

demershov opened the pull request:
1. Добавлены новые станции.
2. Исправлены координаты уже существующих.
3. Исправлен баг с обходом не всех станций, т.к в reduce значение station бралось со второго значения.

dternar committed:
Обновление карты и станций

- Добавлены новые станции
- Исправлены координаты существующих
- Иправлен баг с обходом Stations.lenght - 1

philosophocat reviewed (changes requested):
Идея второй версии карты - удешевление её актуализации. Как только обновляется карта на официальном сайте метро, мы просто копируем её (http://mosmetro.ru/bitrix/templates/sh1/map/new_map.svg), а gulp-таск вычищает карту от ненужных данных и упаковывает код и графику воедино. Мы добавляем для новых станций координаты зелёной иконки для выделенного состояния и проверяем, что всё работает корректно. Исходя из этой идеи, содержимое map.svg всегда ровно такое же, как на сайте метро. Поправьте, пожалуйста, этот момент и я приму ваши изменения.

dternar committed:
Update map

1. Исправлена карта и пересобран проект.
2. В таске галпа добавлено регулярное выражение для комментариев.
